### PR TITLE
data: gawadeatul

### PR DESCRIPTION
**Pull request by gawadeatul**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4870"></a>

### CODE REVIEW

**Review by md-abid-hussain — APPROVED** [review]
LGTM

File: data/gawadeatul.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Atul Gawade",
+	"displayStatsPublic": "true",
```
Comment (imbeshat) [inline data/gawadeatul.json:3]:
```suggestion
	
```

File: data/gawadeatul.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	"type": "personal",
+	"bio": "I am Backend Developer",
+	"avatar": "https://github.com/gawadeatul.png",
```
Comment (imbeshat) [inline data/gawadeatul.json:6]:
```suggestion
	
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Review by imbeshat — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**